release/images.sh

```diff
@@ -19,6 +22,23 @@ LOG_DIR="${WORK_DIR}/logs"
 PROCS_PER_IMG=8
 TEMPLATES_PATH=${TEMPLATES_PATH:-"${PWD}/config/images"}
 
+sign_checksum() {
+    chksum_file=$1
+
+    if [[ ! -f "${IMG_SIGN_KEY}" && -z "${IMG_SIGN_CMD}"  ]]; then
+        warn "Skip signing due to no signing command nor signing key provided"
+        exit 0
```

Comment:
Do you mean `return`? We dont want to quit the entire script

Reply:
my bad copy and paste.

Reply:
Fixed.